cloud-aws-cloudformation/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/AwsCloudWatchService.java

```diff
@@ -70,7 +70,8 @@ public void addCloudWatchAlarmsForSystemFailures(List<CloudResource> instances,
         }).forEach(instance -> {
             try {
                 PutMetricAlarmRequest metricAlarmRequest = new PutMetricAlarmRequest();
```

Comment:
General note: There is a separate `AwsNativeCloudWatchService`. Would not it be better to reuse / modify that one instead?

Reply:
I fear `AwsNativeCloudWatchService.ALARM_ACTION_PREFIX` will be still wrong:
```
private static final String ALARM_ACTION_PREFIX = "arn:aws:automate:";
```